lib/rubocop/cop/cop.rb

```diff
@@ -246,6 +236,23 @@ def external_dependency_checksum
         nil
       end
 
+      ### Deprecated registry access
+
+      # Deprecated. Use Registry.global
```

Comment:
Might be a good idea to put some YARD annotations on the deprecated methods.

Reply:
Thanks, didn't know there was a special tag